pkg/operator/operator.go

```diff
@@ -394,9 +394,9 @@ func (optr *Operator) sync(key string) error {
 		// "RenderConfig" must always run first as it sets the renderConfig in the operator
 		// for the sync funcs below
 		{"RenderConfig", optr.syncRenderConfig},
+		{"MachineConfigController", optr.syncMachineConfigController},
```

Comment:
Hmm, just trying to remember the inter-dependencies here. Other than renderconfig being first and requiredpools being last, were MCP/MCD/MCC interdependent on each other? How does doing MCC first help here?

Reply:
yeah, https://github.com/openshift/machine-config-operator/commit/df7ff4d74e9f48e3b52660f94bea648ef571e26a . If possible we should maintain sync ordering to avoid any undesired behavior.